openpype/modules/deadline/deadline_module.py

```diff
@@ -10,7 +10,15 @@ def initialize(self, modules_settings):
         # This module is always enabled
         deadline_settings = modules_settings[self.name]
         self.enabled = deadline_settings["enabled"]
-        self.deadline_url = deadline_settings["DEADLINE_REST_URL"]
+        deadline_url = deadline_settings.get("DEADLINE_REST_URL")
+        if not deadline_url:
+            deadline_url = deadline_settings.get("deadline_urls", {}).get("default")  # noqa: E501
+        if not deadline_url:
+            self.enabled = False
+            self.log.warning(("default Deadline Webservice URL "
+                              "not specified. Disabling module."))
+            return
+        self.deadline_url = deadline_url
 
     def get_global_environments(self):
```

Comment:
Does make sence that deadline is setting this global environment if is not used?